string-manipulation-1/src/is-vowel.js

```diff
@@ -0,0 +1,20 @@
+/* exported isVowel */
+function isVowel(character) {
+  switch (character) {
+    case 'A':
+    case 'E':
+    case 'I':
+    case 'O':
+    case 'U':
+    case 'Y':
+    case 'a':
+    case 'e':
+    case 'i':
+    case 'o':
+    case 'u':
+    case 'y':
+      return true;
+    default:
+      return false;
+  }
+}
```

Comment:
If you convert `character` to lowercase, then you don't need to check lowercase and uppercase.
```
function isVowel(character) {
  switch (character.toLowerCase()) {
    case 'a':
    case 'e':
    case 'i':
    case 'o':
    case 'u':
      return true;
    default:
      return false;
  }
}
```